docs/TROUBLESHOOTING.md

```diff
@@ -276,6 +276,37 @@ This section includes how to debug a failed restore. For more specific issues re
 
 <hr style="height:1px;border:none;color:#333;">
 
+<h3 align="center">Restic Restore Partially Failing on OCP 4.14 Due to Changed PSA Policy<a id="resbackup"></a></h3>
```

Comment:
add a link after line num 17 so it's in the list please.
This is looks good, thanks for adding it.